mitosheet/mitosheet/streamlit/v1/spreadsheet.py

```diff
@@ -108,6 +108,17 @@ def get_function_from_code_unsafe(code: str) -> Optional[Callable]:
         
     raise ValueError(f'No functions defined in code: {code}')
 
+# This is the class that is returned when the user sets return_type='analysis'
+# It contains data that could be relevant to the streamlit developer, and is 
+# used for replaying analyses. 
+class MitoAnalysis:
+    code: str
+    code_options: CodeOptions
```

Comment:
I'm actually like -- why don't we make all of this private for now? If we want to expose these things publicly, we can always make accessors. But I think keeping it private for now is sensible - so we don't have to worry about compatibility.